data/xsum/facebook-bart-large-xsum-summaries.json

```diff
@@ -1,22 +1,313 @@
 {
   "57f7b78a61786d1f4d3c668d10c991ad": {
     "35345872": {
-      "summary": [
-        "Pakistan has lifted its ban on YouTube, four years after it was banned over a blasphemous video."
-      ],
-      "metadata": {}
+      "summary": "Pakistan has lifted its ban on YouTube, four years after it was banned over a blasphemous video.",
+      "metadata": {
+        "tokens_with_entropy": [
```

Comment:
Unsure if this is the best format to store 🤔 

Perhaps we can store token ids instead and invoke the tokenizer in the view to decode them

Reply:
@cephcyn  / @srush / @vincehartman38  do you have any intuition for what we should store here? Are you OK with this format?

A tuple of the token + per-token-entropy seems excessive since we've already stored the full summary. However on the other hand it seems like the best way to be explicit about which token the entropy belongs to? We could store token id + entropy, but that requires loading the tokenizer in the view

Downstream potential use-cases would be:
- visualize entropy at the token level in a selected summary
- visualize/explore patterns of per-token-entropy across all summaries

Reply:
@anton164 Can you give an example of what the cache would look like with the entropy and token data. Curious to see what it would look like.